requirements.in

```diff
@@ -1,5 +1,6 @@
-aiohttp>=3.8.1
-attrs>=20.3.0
-cryptography>=36.0.1
-kubernetes>=12.0.1
-typedload==2.7
+aiohttp
```

Comment:
We can still have > in the `requirements.in` since I assume we are not fine with older versions of these libraries.